packages/cli/src/workflows/workflows.services.ee.ts

```diff
@@ -230,7 +231,7 @@ export class EEWorkflowsService extends WorkflowsService {
 				return;
 			}
 			Object.keys(node.credentials).forEach((credentialType) => {
-				const credentialId = parseInt(node.credentials?.[credentialType].id ?? '', 10);
+				const credentialId = node.credentials?.[credentialType].id ?? '';
```

Comment:
No context here but if there is no ID we should return early instead of trying to find a match for `''`?